app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -13,6 +13,10 @@ def self.binding_address
       BINDING_ADDRESS
     end
 
+    def self.preload_for_console
+      configure_secret_token(SecureRandom.hex(64))
+    end
+
```

Comment:
@matthewd Thanks for the idea, I like how the two consumers of `configure_secret_token`, line 17 and 26 are next to each other.